CRM/Financeexportformats/Hook/AlterContent/FinancialExportForm.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class CRM_Financeexportformats_Hook_AlterContent_FinancialExportForm {
+
+  private $pageContent;
+
+  public function __construct(&$content) {
+    $this->pageContent = &$content;
+  }
+
+  public function run() {
+    $this->fixButtonDesignWithInlineCss();
+  }
+
+  private function fixButtonDesignWithInlineCss() {
```

Comment:
I think a better name would be:

fixFormButtonsAlignment()

like that I can tell it the code is about fixing the buttons alignment, if I need to know "how" it does that, I can just read the method code.

Reply:
Update the function name.